packages/ocsigen-toolkit/ocsigen-toolkit.2.13.0/opam

```diff
@@ -0,0 +1,26 @@
+opam-version: "2.0"
+maintainer: "dev@ocsigen.org"
+synopsis: "Reusable UI components for Eliom applications (client only, or client-server)"
+description: "The Ocsigen Toolkit is a set of user interface widgets that facilitate the development of Eliom applications."
+authors: "dev@ocsigen.org"
+homepage: "http://www.ocsigen.org"
+bug-reports: "https://github.com/ocsigen/ocsigen-toolkit/issues/"
+dev-repo: "git+https://github.com/ocsigen/ocsigen-toolkit.git"
+license: "LGPL-2.1-only with OCaml-LGPL-linking-exception"
+build: [ make "-j%{jobs}%" ]
+install: [ make "install" ]
+depends: [
+  "ocaml" {>= "4.08.0"}
+  "eliom" {>= "6.12.1"}
+  "calendar" {>= "2.0.0"}
+]
+depexts: [
+  ["libgdbm-dev"] {os-family = "debian"}
```

Comment:
Is that a hack to help opam 2.0's depext plugin?
2.1 works perfectly and it doesn't seem to help opam 2.0 either.

I would suggest to remove this or if you need dbm explicitly (i'm guessing it's not the case) use the `conf-dbm` package.

Reply:
Thanks. It seems that this dependency was either added by accident or to resolve some obscure error with GitHub Actions. Something like that...